Modules/Scripted/ExtensionWizard/ExtensionWizard.py

```diff
@@ -206,13 +209,13 @@ def setupTemplates(self):
                     self.templateManager.addCategoryPath(c, path)
                 except:
                     mp = (c, path)
-                    qt.qWarning("failed to add template path %r for category %r" % mp)
+                    qt.qWarning(_("failed to add template path %r for category %r") % mp)
                     qt.qWarning(traceback.format_exc())
 
     # ---------------------------------------------------------------------------
     def createExtension(self):
-        dlg = CreateComponentDialog("extension", self.parent.window())
-        dlg.setTemplates(self.templateManager.templates("extensions"))
+        dlg = CreateComponentDialog(_("extension"), self.parent.window())
+        dlg.setTemplates(self.templateManager.templates(_("extensions")))
```

Comment:
I don't think is is allowed to be translated. Please test.
```suggestion
        dlg.setTemplates(self.templateManager.templates("extensions"))
```